pages-e2e/features/_utils/getAssertVisible.ts

```diff
@@ -38,4 +39,5 @@ async function assertVisible(
 			: ''
 	}`;
 	assert(isVisible, `expected ${elementStr} to be visible but it isn't`);
+	return locator;
```

Comment:
Why not simply `return locator;` at the point where you `break` out of the loop? Then you don't need to define the `locator` outside the loop, and also you can just immediately throw an error if you exit the loop rather than testing again for visibility.

Reply:
ah yeah good point! 🙂👍

Reply:
code updated 🙂